src/runtime/internal/transitions.ts

```diff
@@ -79,6 +79,9 @@ export function transition_out(block: Fragment, local: 0 | 1, detach?: 0 | 1, ca
 		});
 
 		block.o(local);
+		// fix: destroy non-fragment element such as empty components.
```

Comment:
This is super nits, but such comment can recognize by Git log and test so I don't think we need it here.

Reply:
Agree.